src/utils/hints/namesHints.js

```diff
@@ -0,0 +1,14 @@
+export const namesHints = {
+  name: 'Unique identifier that may be configured to point to accounts, contracts or oracles.',
+  nameInAuction: 'Unique identifier that may be configured to point to accounts, contracts or oracles. Names shorter than 13 characters need to be auctioned.',
+  expires: 'Estimated date, time, and block height when the name will be detached from the account.',
```

Comment:
I would change it to:
"Block height and estimated date and time when the name will be detached from the account."

Reply:
I fixed it here. The rest I suggest to fix at once https://github.com/aeternity/aescan/issues/307